TypeCobol.Test/Parser/Programs/TypeCobol/ProcedureInput.rdz.tcbl

```diff
@@ -30,11 +32,34 @@
            .
        END-DECLARE.
       
+       declare procedure Foo input var1 pic X.
+       procedure division.
+      * Ok, 1st param is also input in Foo2
+           call Foo2 input var1
+      * Ko, 1st param is also in-out in Foo3
+           call Foo3 in-out var1
+           goback
+           .
+       end-declare.
+      
+       declare procedure Foo2 public
+           input var1 pic X.
+       procedure division.
+           goback.
+       end-declare.
+      
+       declare procedure Foo3 public
+           in-out var1 pic X.
+       procedure division.
+           goback.
+       end-declare.
+      
        Traitement.
            call IsPersonValid
                 INPUT myPerson, myName
                 IN-OUT myNbr
                 OUTPUT result
+                 call Foo input myvar1
```

Comment:
indent ?

Reply:
I unresolved this comment because it seems the indentation is still inconsistent. Is it on purpose ?